### PR TITLE
Dep bumps/pins and doc tidy

### PR DESCRIPTION
**Pull request by xross**
* Dep bumps & pins:
   - lib_xua: develop -> 5.0.0
   - lib_i2c: 6.2.0 -> 6.4.0
   - lib_i2s: 5.1.0 -> 6.0.1
   - lib_board_support: 1.0.0 -> 1.1.1
* Readme updates
* Bump XJSL 0.34 -> 0.35
* Bump xmosdoc 6.1.3 -> 6.2.0
* Doc tidies
* Semi-automatic changelog fixes
* Modify archiving to retain xe files

### CODE REVIEW

File: doc/rst/app_316_mc.rst [hunk 1]
```diff
@@ -116,14 +116,14 @@ analogue input. Configuration of both the DAC and the ADC takes place over I²C.
 Configuring audio hardware
 --------------------------
 
-All of the external audio hardware is configured using ``lib_board_support``.
+All of the external audio hardware is configured using `lib_board_support <http://www.xmos.com/file/`.
```
Comment (danielpieczko) [inline doc/rst/app_316_mc.rst:119]: The end of the URL is missing

File: doc/rst/drivers.rst [hunk 2]
```diff
@@ -2,28 +2,32 @@
...
     - Since version 10, release 1809, Windows natively supports USB Audio Class 2.0
 
-Thirds Party Windows Drivers
-============================
+Third party Windows drivers
+===========================
 
-For some products it may be desirable to use a third-party driver for Windows. A number reasons exist as to why this may be desirable:
+For some products it may be desirable to use a third-party driver for Windows.
+A number reasons exist as to why this may be desirable:
```
Comment (danielpieczko) [inline doc/rst/drivers.rst:30]:
```suggestion
A number of reasons exist as to why this may be desirable:
```

**Review by danielpieczko — APPROVED** [review]
A couple of comments but otherwise looks good

**Comment by xross** [comment]
@danielpieczko I fixed the minor doc issues, thanks. I also modified the archiving to keep .xe files.

**Pull request merged**